### PR TITLE
Add RBI for IRB

### PR DESCRIPTION
**Pull request by Morriar**
No signatures.

### CODE REVIEW

**Comment by elliottt** [comment]
Internal stripe build: http://go/builds/bui_HHhg4Wr38fP6ZY

**Comment by Morriar** [comment]
This one needs https://github.com/sorbet/sorbet/pull/3026 to be merged first.

**Comment by Morriar** [comment]
Rebased on #3026 and updated the `constant-fuzzy.out` file

**Comment by jez** [comment]
http://go/builds/bui_HJZMgD8ZKbJRXD

File: rbi/stdlib/weakref.rbi [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+#
+# ```ruby
+# foo = Object.new            # create a new object instance
+# p foo.to_s                  # original's class
+# foo = WeakRef.new(foo)      # reassign foo with WeakRef instance
+# p foo.to_s                  # should be same class
+# GC.start                    # start the garbage collector
+# p foo.to_s                  # should raise exception (recycled)
+# ```
+class WeakRef
```
Comment (jez) [inline rbi/stdlib/weakref.rbi:18]: When I tested this on Stripe's CI, it said that `WeakRef` descends from `Delegator`, which i confirmed via IRB:

```
❯ irb
irb(main):001:0> require 'weakref'
=> true
irb(main):002:0> WeakRef.ancestors
=> [WeakRef, Delegator, #<Module:0x00007fcfee88a8e0>, BasicObject]
```

```suggestion
class WeakRef < Delegator
```

**Pull request merged**